azure-pipelines.yml

```diff
@@ -112,8 +112,8 @@ jobs:
 
 - template: eng/pipelines/test-unix-job.yml
   parameters:
-    testRunName: 'Test OSX Debug'
-    jobName: Test_OSX_Debug
+    testRunName: 'Test macOS Debug'
+    jobName: Test_macOS_Debug
```

Comment:
What is the distinction in the change here?

Reply:
OS X was renamed to macOS 4 years ago at this point. It's one of those things I can't help myself from changing whenever I see it.

Reply:
So my knowledge is 4 years out of date then :smile: 

Sounds good.